theme/static/css/hydroshare_core.css

```diff
@@ -1604,10 +1604,6 @@ div#sharing-status {
     overflow: hidden; /* Fixes height for floating elements */
 }
 
-.info-group {
-	width: calc(100% - 220px);
-}
-
```

Comment:
The part I removed is no longer needed. I will point the rule directly where its needed now instead of using it as a container.